Add RW1990 support. (#4)
toitware/toit-ibutton: README.md

Typically, the readers have two wires, one for data/power and one for ground. The
1-wire protocol expects the data wire to be pulled up to power by a resistor.

In most cases, the pull-up of the ESP32 is strong enough to power devices that
are connected to the 1-wire bus. As such, no additional components are needed.

See https://pdfserv.maximintegrated.com/en/an/AN4255.pdf for more information on
how to power 1-wire devices.

## RW1990 support

> The RW1990 is a device that allows to modify the ID of an iButton. It is not

Reviewer: ```suggestion
The RW1990 is a device that can modify the ID of an iButton. It is not
```

Author: It's not the RW1990 that can modify the ibuttons, but the RW1990 is an iButton device with a changeable ID.
I updated the description.
done.

produced by Dallas, but can be found on Chinese websites.

We were not able to reliable change all RW1990 devices we received. Out of

> 10 devices, 4 worked without issues, 4 did not allow to change the IDs at all,

Reviewer: ```suggestion
10 devices, 4 worked without issues, 4 did not allow changing the IDs at all,
```
